doc/source/changes/version_0_28.rst.inc

```diff
@@ -232,7 +232,28 @@ Miscellaneous improvements
 
     >>> arr = read_csv('arr.csv', nb_axes=3)
 
-  Closes :issue:`548`:
+  Closes :issue:`548`.
+
+* added the relative tolerance `rtol` and the absolute tolerance `atol` arguments to the `LArray.equals` method.
+  These two arguments can used to test the equality of two floating arrays within a given relative and/or absolute
+  tolerance:
+
+    >>> arr1 = LArray([0.9999, 1.0001], "a=a0,a1")
+    >>> arr1
+    a      a0      a1
+       0.9999  1.0001
+    >>> arr2 = ones("a=a0,a1")
+    >>> arr2
+    a   a0   a1
+       1.0  1.0
+    >>> arr1.equals(arr2, atol=5e-5)
+    False
+    >>> arr1.equals(arr2, rtol=5e-5)
```

Comment:
the fact that array2 is 1 makes rtol not very interesting.

I would go for something like:
```python
arr1 = LArray([10, 10], "a=a0,a1")
arr2 = LArray([9.99, 10.01], "a=a0,a1")
```